### DISCUSSION TITLE
Retail SDK to Commerce SDK migration

### DISCUSSION THREAD

**Discussion by meerrahi** [microsoft/Dynamics365Commerce.Solutions#217] [Q&A]
My client currently operates over 200 stores, all of which are equipped with MPOS installations. Customizations have been implemented on both the MPOS side and the server side using Retail SDK. Presently, we are in the process of migrating from Retail SDK to Commerce SDK. I have a concern regarding the deployment of Commerce SDK on the production CSU: will the existing MPOS installations continue to function seamlessly? Given the substantial number of devices, deploying the Commerce app to all of them would be a time-consuming task. Therefore, it is imperative that the business remains operational with the old MPOS installations until the new Commerce app is successfully installed.

**Comment by shahbazchishty** [comment]
The Retail SDK used Retail Server which has been replaced by the CSU in the Commerce SDK. I don't think there should be a problem for existing MPOS installation to use the CSU. But for production, I would still suggest to have it tested in your test/dev environment.
